diff --git a/node_modules/graphql/jsutils/instanceOf.mjs b/node_modules/graphql/jsutils/instanceOf.mjs
index 0f05267..e8776d6 100644
--- a/node_modules/graphql/jsutils/instanceOf.mjs
+++ b/node_modules/graphql/jsutils/instanceOf.mjs
@@ -9,7 +9,7 @@ import { inspect } from './inspect.mjs';
 export const instanceOf =
   /* c8 ignore next 6 */
   // FIXME: https://github.com/graphql/graphql-js/issues/2317
-  globalThis.process && globalThis.process.env.NODE_ENV === 'production'
+  globalThis.process && process.env.NODE_ENV === 'production'
     ? function instanceOf(value, constructor) {
         return value instanceof constructor;
       }
